### ISSUE TITLE
[Feature request] IPTV playlist auto pooling

### ISSUE THREAD

**Issue by egorvas** [AndreyPavlenko/Fermata#270]
Hi Andrey, I am using Fermata app for watching Twitch via mu38 playlist (did the special simple server for it https://github.com/egorvas/twitch-playlist-server)
But m3u8 links for twtitch updating dynamically, so I need to remove and add playlist every time when I want to start watching.
I will be nice to have feature for auto pooling m3u8 configuration periodically if m3u8 config added with a url.

**Comment by AndreyPavlenko** [comment]
The file is currently cached and updated no earlier than 24 hours later. Probably, some option should be added for the cache max_age.

**Comment by egorvas** [comment]
Yes it will be perfect to have max_age as options variable, cause in my case playlist file needs to be updated very max_age.
